MSSQL/OMOPLoader.sql

```diff
@@ -1823,7 +1825,9 @@ inner join visit_occurrence enc on enc.person_id = m.patient_num and enc.visit_o
 inner join (select * from i2o_ontology_lab where i_stddomain='LOINC') lab on lab.c_basecode  = M.concept_cd
 inner join i2o_mapping omap on lab.i_stdcode=omap.source_code and omap.domain_id='Measurement'
 --left outer join pmn_labnormal norm on ont_parent.c_basecode=norm.LAB_NAME
+-- NOTE: Both m.units_cd (original observation fact unit value) and m.i_unit (extract unit value from PHS XML) are mapped to UCUM standard concepts
 left outer join i2o_unitsmap u on u.units_name=m.units_cd
+left outer join i2o_unitsmap u2 on u.units_name=m.i_unit

```

Comment:
I think it'd be cleaner to use one left join with and OR in the ON clause?

Reply:
Per our discussion on Friday leaving this as 2 left outer joins with the isnull() function in the select provide more control over precedence and avoids accidental duplicate records.